### PR TITLE
Feat/responsive

### PR DESCRIPTION
**Pull request by julialoeschel**
- mage pages responsive

### CODE REVIEW

File: src/app/components/YourLocationInput/YourLocationInput.tsx [hunk 1]
```diff
@@ -35,6 +35,9 @@ const Container = styled.div`
     'text text'
     '. location1'
     '. location2';
+  @media only screen and (min-width: 640px) {
+    grid-template-columns: 20% 80%;
+  }
```
Comment (Torfeltorfel) [inline src/app/components/YourLocationInput/YourLocationInput.tsx:40]: cool!

**Review by Torfeltorfel — APPROVED** [review]
toll!

File: src/app/pages/Map/Map.tsx [hunk 2]
```diff
@@ -454,7 +454,10 @@ const LocationInput = styled.div`
 
   & .mapboxgl-ctrl-geocoder--icon-search {
     margin-top: -0.3em;
-    margin-left: 15em;
+    margin-right: 15em;
+    right: 12px;
```
Comment (janhendrikstruve) [inline src/app/pages/Map/Map.tsx:458]: Maybe use a consistent measuring type? 🦆

**Review by janhendrikstruve — APPROVED** [review]
LGTM !!! 🐮 just one liiiiittle thing

**Pull request merged**